AndroidPubSub/build.gradle

```diff
@@ -18,7 +18,7 @@ repositories {
 }
 
 dependencies {
-    def aws_version = "2.12.+"
+    def aws_version = "2.13.+"
     implementation "com.amazonaws:aws-android-sdk-iot:$aws_version"
```

Comment:
This probably also needs to change to AAR

Reply:
This is not needed anymore with the version of gradle we are on and should probably be removed from other places as well.